app/app/models.py

```diff
@@ -0,0 +1,22 @@
+from . import db
+from flask_login import UserMixin
+from sqlalchemy.sql import func
+from werkzeug.security import generate_password_hash, check_password_hash
+
+class User(db.Model, UserMixin):
+    id = db.Column(db.Integer, primary_key=True)
+
+    username = db.Column(db.String(64), index=True, unique=True)
+    password_hash = db.Column(db.String(128))
+    testAttempts = db.relationship('TestAttempt', backref = "student", lazy = "dynamic")
```

Comment:
test attempts?, i don't think we need this atm

I have created the ERD and posted the link in #3

Reply:
yep agreed, thats a relic from my previous project i didn't catch

Reply:
I have removed this in #32  (while I was setting it up because the migration scripts are failing that made me unable to create the database)

Reply:
so I think all good for this one